### ISSUE TITLE
Mangahere downloads failing?

### ISSUE THREAD

**Issue by Sorris2018** [riderkick/FMD#1101]
Mangahere downloads always showing as a red line, looks like the manga's actually complete, (I have a directory with the images in it) but it's not zipping on completion, and is leaving me with a red line in the download window.

**Comment by Tmp341** [comment]
Have you checked that your files and mangahere files are exact same? I mean, downloaded chapter contains 15 pages on mangahere and yours has 15 pages too.

**Comment by Sorris2018** [comment]
Yeah, for example, I just checked the mangehere site (which is apparently infected as hell) and there's 59 pages for the most recent chapter of magikano, and it downloaded 59 pages, but never marked it complete.

**Comment by Sorris2018** [comment]
Seems like if you keep resuming the task, eventually it finishes. then it'll sometimes only zip 1 page and leave all the rest in a separate folder. Which is odd.

**Comment by Kilvearn** [comment]
Same problem here... It can be solved by setting "Number of downloaded files per task at the same time" to "1", but it's very slow then...

**Comment by Tmp341** [comment]
They most likely blocking high speed downloads. I think slow speed is better then being banned.

**Comment by Kilvearn** [comment]
I tried with many sites (MangaHere, MangaReader and MangaFox), same problems for all of them :-/
